.NET/Microsoft.Recognizers.Text.Sequence/Parsers/CreditCardParser.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Linq;
+using System.Text.RegularExpressions;
+using Microsoft.Recognizers.Definitions;
+
+namespace Microsoft.Recognizers.Text.Sequence.Parsers
+{
+    public class CreditCardParser : BaseSequenceParser
+    {
+
+        // @TODO move regexes to base resource files
```

Comment:
Parser should figure out the issuer/etc, crc, and filter out false positive matches.